### ISSUE TITLE
the keras's dropout and regularization is much slower than lasagne?

### ISSUE THREAD

**Issue by fayeshine** [fchollet/keras#1714]
In lasagne, sknn, I add dropout and regularization, the training speed didnt gain too much. But in keras, it becoms quite slow, several times than before.
Is there anything wrong?

**Comment by farizrahman4u** [comment]
Do you have any code that would help us reproduce the issue?

**Comment by farizrahman4u** [comment]
Do you have any code that would help us reproduce the issue?
